examples/grafana-dashboard/main.tf

```diff
@@ -1,22 +0,0 @@
-resource "chronosphere_grafana_dashboard" "my_dashboard" {
```

Comment:
should we keep this example with the new classic terminology?

Reply:
there is already an example for classic dashboards. The only difference is that it uses a collection instead of a bucket for a container.